setup.py

```diff
@@ -528,6 +528,8 @@ def finalize_options(self):
                 "onnxruntime.training.ortmodule.torch_cpp_extensions.cpu.torch_interop_utils",
                 "onnxruntime.training.ortmodule.torch_cpp_extensions.cuda.torch_gpu_allocator",
                 "onnxruntime.training.ortmodule.torch_cpp_extensions.cuda.fused_ops",
+                "onnxruntime.training.ortmodule.ort_triton",
```

Comment:
Do we need package triton added as a requirement?

Reply:
Not quite familiar with this part, but I see that all folders with __init__.py are added here so I follow the same way...